teamC/choojaeho/app.js

```diff
@@ -0,0 +1,187 @@
+    const {name, password, email, profile_image} = req.body;
+
+    await database.query(
+        `INSERT INTO users (
+            name,
+            password,
+            email,
+            profile_image
+        ) VALUES (?, ?, ?, ?);
+        `,  
+        [name, password, email, profile_image]
+    )
+    res.status(201).json({message : "user created"})
+});
+//과제3번
+app.post("/postsman", async (req, res, next) => {
+    const {title, content, user_id} = req.body;
+
+    await database.query(
+        `INSERT INTO posts (
+            title,
+            content,
+            user_id
+        ) VALUES (?, ?, ?);
+        `,  
+        [title, content, user_id]
+    )
+    res.status(201).json({message : "postCreated"})
+});
+//과제 4번
+app.get("/postman", async (req, res ) => {
+
+    const user = await database.query(
+        `
+            SElECT 
+            posts.user_id as userId, 
+            users.profile_image as userProgileImage, 
+            posts.id as postingId, 
+            posts.posts_image as postingImageUrl, 
+            posts.content as postingContent
+            from users join posts on users.id=posts.user_id
+        `
+    )
+
+    res.status(201).json({data : user})
+});
+
+//과제 5번
+app.get("/posts/:userId", async (req, res, next) => {
+    const userId = req.params.userId;
+    
+     await database.query(
+        `
+            SELECT
+                users.id as userId,
+                users.name as userName
+                from users where users.id =${userId}
+        `,
+            (err, data) => {
+                    database.query(
+                    `
+                        SELECT
+                        posts.id as PostingId, 
+                        posts.title as PostingName, 
+                        posts.content as PostingContent
+                        FROM posts
+                        where posts.user_id = ${userId}
+                    `,(err, posting) => {
+                        data[0]["postings"] = posting
+                        res.status(200).json({"data":data[0]})
+                    }
+        )}
+    )
+});
+
+//6번
+app.patch("/updates/:nextPosts", async (req, res, next) => {
+    const userId = req.params.nextPosts;
+    const {modified} = req.body
+    
+    await database.query(
+        `UPDATE posts
+         SET content = ?
+         WHERE user_id = ${userId}
+        `,
+        //? 자리에 업데이트값이 들어간다!!!!!!!!!!!!!!!
+        [modified] );
+    await database.query(
+        `SELECT
+        users.id as userId,
+        users.name as userName,
+        posts.id as PostingId, 
+        posts.title as PostingName, 
+        posts.content as PostingContent
+        FROM users,posts
+        WHERE posts.user_id=${userId} 
+        `,
+        (err, data) => {
+            console.log(data);
+            res.status(201).json({"data":data[0]})
+        }
+    )
+});
+
+
+
+const PORT = process.env.PORT;
+const server = http.createServer(app);
```

Comment:
해당 코드는 line 182와 183 사이에 들어가면 좋을 것 같습니다.

```suggestion
const PORT = process.env.PORT;
const server = http.createServer(app);
```

Reply:
<img width="600" alt="스크린샷 2022-09-06 오후 5 01 34" src="https://user-images.githubusercontent.com/53214510/188580083-194128c0-2236-4ffc-8714-86b4e7a5e48a.png">